src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
-export interface Figure {
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+function getRound(num: number): number {
+  return Math.floor(num * 100) / 100;
 }
 
-export class Triangle {
-
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  constructor(
+    public color: Color,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const sides = [a, b, c].sort((d, e) => d - e);
+
+    if (sides[2] >= sides[0] + sides[1]
+        || sides.some((side) => side <= 0)
```

Comment:
try to move it to the separate variable with meaningful name, otherwise other developers will have hard times to understand what is this condition about